### ISSUE TITLE
Could you please explain the data format of relation prompts?

### ISSUE THREAD

**Issue by c-box** [yanaiela/pararel#1]
For example:
```
{"pattern": "[X] originated from [Y].", "lemma": "originate", "extended_lemma": "originate-from", "tense": "past"}
```

What's the meaning of each keyword？

**Comment by yanaiela** [comment]
Hi @c-box

Here it is:

`pattern`: The pattern we created for describing the different relations, between a subject ([X]) and an object ([Y]), which are placeholders, and replaced in practice with a a subject from a dataset (e.g. LAMA) and a masked token that needs to be predicted (e.g. [MASK])
`lemma` and `extended_lemma`: these are our attempt to obtain the core lemma/verb in the pattern. We used them internally to determine which `extended_lemma` entail each other, thus taking cases where both lemmas entail each other, thus making them paraphrases.
`tense`: is the tense of the pattern. If I recall correctly we included `future`, `present`, `past` and `?` which means that the pattern can be read in context with more than one tense.
